curriculum/challenges/_meta/workshop-reusable-mega-navbar/meta.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Reusable Mega Navbar",
```

Comment:
the name will need to change here too to match the convention already laid out in the other workshops.